src/Storage/RedisAdapter.php

```diff
@@ -179,7 +179,14 @@ private function collectGauges(): array
             $this->redis->multi();
             $this->redis->hGetAll($metricKey);
 
-            if(in_array($metricKey, config('prometheus.metrics_to_delete'), true)) {
+            if (
+                !is_array(config('prometheus.metrics_do_not_delete'))
+                || (
+                    is_array(config('prometheus.metrics_do_not_delete'))
+                    && !in_array($metricKey, config('prometheus.metrics_do_not_delete'), true)
```

Comment:
Add this param to config example

Reply:
Lets simplify if clause: `!in_array($metricKey, config('prometheus.metrics_do_not_delete', []), true)` make assumption that
prometheus.metrics_do_not_delete is array anyway